src/config/example.toml

```diff
@@ -182,3 +182,4 @@ backup_storage_url = "memory"
 backup_storage_directory = "backup"
 max_concurrent_creating_streaming_jobs = 1
 pause_on_next_bootstrap = false
+wasm_storage_url = "fs://.risingwave/data"
```

Comment:
I have no idea what the default path should be in a production environment. 🤔

Reply:
default to none?

Reply:
The path `.risingwave/data` should be specific to RiseDev. 🤔 Maybe you want to set it with `risedev d`?